pkg/ocm/accounts_mgmt.go

```diff
@@ -22,17 +22,18 @@ const (
 
 //go:generate mockgen -package ocm -destination mock_accounts_mgmt.go . OCMAccountsMgmt
 type OCMAccountsMgmt interface {
-	CreateSubscription(ctx context.Context, clusterID strfmt.UUID) (*amgmtv1.Subscription, error)
+	CreateSubscription(ctx context.Context, clusterID strfmt.UUID, clusterName string) (*amgmtv1.Subscription, error)
 	GetSubscription(ctx context.Context, subscriptionID strfmt.UUID) (*amgmtv1.Subscription, error)
-	UpdateSubscription(ctx context.Context, subscriptionID, openshiftClusterID strfmt.UUID) (*amgmtv1.Subscription, error)
+	UpdateSubscriptionPostInstallation(ctx context.Context, subscriptionID, openshiftClusterID strfmt.UUID) (*amgmtv1.Subscription, error)
```

Comment:
why do we actually need to add the 'PostInstallation' postfix? I mean, from the api view it's still just updating the subscription, right?